_topic_map.yml

```diff
@@ -31,12 +31,23 @@ Topics:
 - Name: About OpenShift Container Engine
   File: oce_about
   Distros: openshift-enterprise
-- Name: Accessing your services
-  File: accessing-your-services
   Distros: openshift-dedicated
```

Comment:
@sheriff-rh just FYI - this distro line needed to be removed as well. I have removed it with: https://github.com/openshift/openshift-docs/pull/18713.

Should this have been CPed to 4.3 branch?

Reply:
Hey @vikram-redhat ! Good catch - I don't think 4.3 was even on the radar when I first opened this PR in October! I CP'ed it just now, all set.

Reply:
Thanks. I will CP the fix there as well.